samples/switchless/README.md

```diff
@@ -103,6 +119,8 @@ called switchlessly at run time. However, this a best-effort directive. Open Enc
 to fall back to a tradition OCALL if switchless call resources are unavailable, e.g., the enclave is not configured
```

Comment:
Not related to your changes, but can you change this to a "traditional OCALL"

Reply:
Fixed.